conf/logback.xml

```diff
@@ -15,7 +15,7 @@
   <logger name="play" level="INFO" />
   <logger name="application" level="DEBUG" />
 
-  <root level="ERROR">
+  <root level="INFO">
     <appender-ref ref="STDOUT" />
```

Comment:
Don't log info to stdout - makes it really hard to rotate files (and if we don't then we could fill up the disk and stop working). Set up a file logger (see flex for an example) and attach the root logger to that.